docs/src/forc_client.md

```diff
@@ -1,3 +1,45 @@
 # forc-client
 
 Forc plugin for interacting with a Fuel node.
+
+## Signing transactions using `forc-wallet` CLI
+
+To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without utxo validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
+
+To install `forc-wallet` please refer to `forc-wallet`'s [github repo](https://github.com/FuelLabs/forc-wallet#forc-wallet).
+
+1. Construct the transaction by using either `forc deploy` or `forc run`. To do so simply run `forc deploy` or `forc run` with your desired parameters. For a list of parameters please refer to [forc-deploy](./forc_deploy) or [forc-run](./forc_run) section of the book. Once you run any of them, you will be asked the address of the wallet you are going to be signing with. After the address is given the transaction will be generated and you will be given a transaction ID. At this point CLI will actively wait for you to insert the signature.
+2. Take the transaction ID generated in the first step and sign it with `forc wallet sign <transaction_id> <account_index>`. This will generate a signature.
+3. Take the signature generated in the second step and provide it to `forc-deploy` (or `forc-run`). Once the signature is provided, the signed transaction will be submitted.
+
+## Initializing the wallet and adding accounts
```

Comment:
The sections for signing and initializing should be flipped. The first section a new user should see is how to initialize rather than how to sign. The signing should then build upon the context of a newly created wallet